github/SecurityVulnerability.py

```diff
@@ -0,0 +1,56 @@
+import github.Commit
+import github.GithubObject
+import github.Package
+
+
+class SecurityVulnerability(github.GithubObject.NonCompletableGithubObject):
+    """
+    This class represents SecurityVulnerability. The reference can be found here in the alert here https://docs.github.com/en/rest/dependabot/alerts#list-dependabot-alerts-for-a-repository
+    """
+
+    def __repr__(self):
+        return self.get__repr__(
+            {
+                "package": self._package.value,
+                "severity": self._severity.value,
+                "vulnerable_version_range": self._vulnerable_version_range.value,
+            }
+        )
+
+    @property
+    def package(self):
+        """
+        :type: :class:`github.Package.Package`
+        """
+        return self._package.value
+
+    @property
+    def severity(self):
+        """
+        :type: string
+        """
+        return self._severity.value
+
+    @property
+    def vulnerable_version_range(self):
+        """
+        :type: string
+        """
+        return self._vulnerable_version_range.value
+
+    def _initAttributes(self):
+        self._package = github.GithubObject.NotSet
+        self._severity = github.GithubObject.NotSet
+        self._vulnerable_version_range = github.GithubObject.NotSet
```

Comment:
You're missing:
- `first_patched_version`

Reply:
intentionally left out but now that I think of it, I will add it.